gusb/meson.build

```diff
@@ -176,7 +176,7 @@ mapfile_target = custom_target('gusb_mapfile',
   output: 'libgusb.ver',
   command: [
     py_installation,
-    join_paths(meson.source_root(), 'contrib', 'generate-version-script.py'),
```

Comment:
about this I would also probably change it to generate_version_script = find_program(meson.source_root() /  'contrib' / 'generate-version-script.py') and then to pass the variable to the command